src/app/cli/src/graphql.ml

```diff
@@ -107,8 +109,8 @@ module Make (Commands : Coda_commands.Intf) = struct
                 | Payment {Payment_payload.Poly.amount; _} ->
                     Ok
                       (amount |> Currency.Amount.to_uint64 |> Stringable.uint64)
-                | Stake_delegation _ ->
-                    (* Stake delegation does not have an amount, so we set it to 0 *)
+                | Stake_delegation _ | Chain_voting _ ->
+                    (* Stake delegation and voting don't have an amount, so we set it to 0 *)
```

Comment:
Since there are commands with no amount, shouldn't the graphql API have it be optional?